.github/workflows/docker_image.yml

```diff
@@ -0,0 +1,70 @@
+---
+name: Docker Image
+
+# https://docs.github.com/en/actions/publishing-packages/publishing-docker-images
+# https://docs.docker.com/build/ci/github-actions/multi-platform
+# https://github.com/opencontainers/image-spec/blob/main/annotations.md
+
+on:
+  push:
```

Comment:
can this match against paths too like existing workflows?

Reply:
I don't know if there is a way to do this since we obviously want the image to be built on tags even if there aren't C++ changes.

One thing to be noticed is that the GitHub Actions cache feature is used here for caching Dockerfile steps, unlike the existing test in linux.yml, so a full rebuild will not happen everytime time and it will be much faster in case only Lua/Markdown are changed.

https://github.com/minetest/minetest/blob/e734b3f0d8055ff3ae710f3632726a711603bf84/.github/workflows/linux.yml#L155-L163

Reply:
I suppose we could match against paths for pull requests, and build on every push.
(add a workflow condition not to tag PRs, obviously)